registry/storage/driver/storagedriver.go

```diff
@@ -110,7 +110,7 @@ type FileWriter interface {
 // number of path components separated by slashes, where each component is
 // restricted to alphanumeric characters or a period, underscore, or
 // hyphen.
-var PathRegexp = regexp.MustCompile(`^(/[A-Za-z0-9._-]+)+$`)
+var PathRegexp = regexp.MustCompile(`^(/[A-Za-z0-9._:-]+)+$`)
```

Comment:
No, we simply don't use schemes and namespaces in tags. This is a horrid hack and not really workable.